Makefile.dune

```diff
@@ -44,6 +48,22 @@ ocheck: voboot
 clean:
 	dune clean
 
+# Dune bootstrap targets:
+
+DUNE_BOOT_VER=1.3.0
```

Comment:
Not so sure I like hard coding the version here. I guess it may get out of sync with the one we use in CI for example.

Reply:
How would you solve that problem? I am not sure we can do better as we are talking about a myriad of different devices envs.

Reply:
write it is a file that is "included" in all the others?